release-notes/opensearch-observability.release-notes-2.10.0.0.md

```diff
@@ -0,0 +1,9 @@
+## Version 2.10.0.0 Release Notes
+
+Compatible with OpenSearch 2.10.0
+
+### Refactoring
+* Fix from upstream core.action changes by @derek-ho in https://github.com/opensearch-project/observability/pull/1590
```

Comment:
The format for release notes doesn't include PR owner's name in it, so can you remove this as well?